source/pages/uv-dose.ipynb

```diff
@@ -0,0 +1,146 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    def astm_g222_21(ghi, sza, m):
</pre><p>Can we rename <code>m</code> to something like <code>mean_coeff</code>?   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AssessingSolar/unofficial-psm3-userguide/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>